projects/step-frontend/src/lib/modules/plan/components/plan-list/plan-list.component.ts

```diff
@@ -53,16 +45,16 @@ export class PlanListComponent {
   }
 
   duplicatePlan(id: string): void {
-    this._httpClient
-      .get<any>(`rest/plans/${id}/clone`)
+    this._plansApiService
+      .clonePlan(id)
       .pipe(
-        map((clone) => {
-          clone.attributes.name += '_Copy';
+        map((clone: Plan) => {
+          clone['attributes']!['name']! += '_Copy';
           return clone;
         }),
-        switchMap((clone) => this._httpClient.post('rest/plans', clone))
+        switchMap((clone) => this._plansApiService.save4(clone))
       )
-      .subscribe((_) => this.dataSource.reload());
+      .subscribe((_) => this._plansApiService.reloadPlansTableDataSource());
```

Comment:
This did not work for me when I tested locally (creating new plan did not show new plan)

Reply:
thanks.
additionally, when importing a plan it does not update as well, but on master!

Reply:
Interesting, I never tried importing. Adding and duplicating updates on the master for me though.